wasm/datatypes/configuration.py

```diff
@@ -0,0 +1,62 @@
+from typing import (
+    NamedTuple,
+)
+
+from wasm.typing import (
+    Store,
+)
+
+from .instructions import (
+    InstructionSequence,
+)
+from .stack import (
+    Frame,
+    FrameStack,
+    Label,
+    LabelStack,
+    ValueStack,
+)
+
+
+class Configuration(NamedTuple):
+    store: Store
+    value_stack: ValueStack
+    label_stack: LabelStack
+    frame_stack: FrameStack
```

Comment:
Should all of these be named with underscore prefix to maintain pseudo privacy?

Reply:
I think you missed the `main.py` module diff (which is collapsed by default)  These are used heavily as a public API on the `config` object.

Reply:
Ah! That's why!